components/brave_wallet_ui/common/ledgerjs/eth_ledger_bridge_keyring.ts

```diff
@@ -0,0 +1,69 @@
+const { EventEmitter } = require('events')
```

Comment:
i think this file is missing a license header?

Reply:
done